src/client/app/containers/BarChartContainer.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import _ from 'lodash';
+import { Bar } from 'react-chartjs-2';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import 'chartjs-plugin-zoom';
+
+/**
+ * @param {State} state
+ */
+function mapStateToProps(state) {
+	const timeInterval = state.graph.timeInterval;
+	const barDuration = state.graph.barDuration;
+	const data = { datasets: [] };
+	// getColor() cycles through the colors, wrapping around the end to the beginning
+	const colors = ['LightBlue', 'GoldenRod', 'Black', 'OrangeRed', 'LightSeaGreen', 'LightSlateGray', 'Purple'];
+	let colorPointer = 0;
+	function getColor() {
```

Comment:
You're doing this in the line chart component too. It should probably get refactored out.